### DISCUSSION TITLE
Is "Colorscheme with Preview" supposed to set the colorscheme when a selection is made.

### DISCUSSION THREAD

**Discussion by mcgbl** [LazyVim/LazyVim#4690] [General]
I don't know what I did to my configuration, but making a selection at "Colorscheme with Preview"  used to set the Colorscheme.
Reading the documentation references manual edit of /nvim/lua/plugins/colorscheme.lua to select a colorscheme but that seems more complicated. Isn't this supposed to work?

**Comment by dpetka2001** [comment]
It does work for me. Selecting a colorscheme from the picker also sets it. I just tried a default LazyVim installation and it works out of the box. Following screencast

[Screencast_2024-11-03-11-09-32.webm](https://github.com/user-attachments/assets/3424592a-860e-4e2a-81ee-f5315df41b22)

**Reply by mcgbl** [reply]
Thank for the reply and taking the time to make a screencast.
I do get the preview just like you do, but exiting nvim and returning defaults back to tokyonight-moon.
Now that I reflect on this issue, I realize I never could set my an alternate Colorscheme I always just kept it at default tokyonight-moon because I liked it best. But now I'm trying to change it.. finding my selection won's stick.

Below is an example of the same result reproduced when I setup LazyVim in a new arch docker container.

1. Setting up the fresh container with arch, gcc, git, neovim, lazyvim. Using default configuration instructions from LazyVimg github.
![241103_15h14m05s_screenshot](https://github.com/user-attachments/assets/fbfadfeb-587b-4a98-a41a-73d4b0bdf867)
1. Running neovim for first time, auto transitions to default tokyo theme as expected.
![241103_15h14m43s_screenshot](https://github.com/user-attachments/assets/c4ce66c1-6428-496b-9653-cfc21765a754)
![241103_15h15m00s_screenshot](https://github.com/user-attachments/assets/271cf033-b3f1-4e8b-967e-cda45032e7c5)
1. Opening ColorScheme with Preview Selector
![241103_15h15m17s_screenshot](https://github.com/user-attachments/assets/2633f3c9-e94b-4d35-8d0a-a663763ff9e5)
![241103_15h15m25s_screenshot](https://github.com/user-attachments/assets/f0b3098c-ff55-4f38-86f0-f19f5fcdbd33)
1. Selecting Colorscheme "blue"
![241103_15h15m40s_screenshot](https://github.com/user-attachments/assets/70c51622-81b5-4228-820d-678108803a23)
1. Colorscheme "blue" selected and back at launch screen, then exiting neovim
![241103_15h15m57s_screenshot](https://github.com/user-attachments/assets/0e260e3e-1e6d-4e39-87a4-2840f7b96391)
1. Relaunching neovim
![241103_15h16m10s_screenshot](https://github.com/user-attachments/assets/616024fc-cbe6-4fe3-bc60-204a32c48136)
1. Colorscheme back to default tokyo
![241103_15h16m23s_screenshot](https://github.com/user-attachments/assets/6790bf34-09d8-4b05-8246-fba55f80a4e1)

**Reply by dpetka2001** [reply]
When you're selecting a colorscheme with Preview it doesn't make it persistent across Neovim restart. You have to do it as it's described in the [docs](https://www.lazyvim.org/configuration#icons--colorscheme) for it to persist.

**Reply by mcgbl** [reply]
Well then. That explains everything then doesn't it?
Thank you for your time.
